src/osd/PG.cc

```diff
@@ -2092,6 +2092,9 @@ void PG::forward_scrub_event(ScrubAPI fn, epoch_t epoch_queued, std::string_view
   if (is_active() && m_scrubber) {
     ((*m_scrubber).*fn)(epoch_queued);
   } else {
+    if (m_scrubber) {
```

Comment:
This should probably be an assert, shouldn't be possible to invoke forward_scrub_event on a destructed PG.

Reply:
done